pytest_fixtures/api_fixtures.py

```diff
@@ -132,13 +133,9 @@ def default_architecture():
     return arch
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope='module')
 def default_os(
-    default_architecture,
-    default_partitiontable,
-    module_configtemaplate,
-    module_provisioingtemplate,
-    os=None,
+    default_architecture, default_partitiontable, module_configtemaplate, os=None,
```

Comment:
`module_configtemaplate` typo `module_configtemplate`

Reply:
Done .. thanks to catch that